### PR TITLE
Updated first para of view-transitions.mdx

### LINKED ISSUE #3780: [CONTENT] - Clarify that <ViewTransitions /> intercepts navigation client-side
### 📚 Subject area/topic

View Transitions

### 📋 Page(s) affected (or suggested, for new content)

https://docs.astro.build/en/guides/view-transitions/

### 📋 General description or bullet points (if proposing new content)

The second opening paragraph of the View Transitions page can appear to imply that the `ViewTransitions` component isn't in fact intercepting the browser's navigation, as the term "client-side routing" is often used to describe intercepting and doing navigation in JS, bypassing the browser's navigation.

> Astro provides a <ViewTransitions /> routing component that can be added to a single page’s <head> to control page transitions as you navigate away to another page, **an effect traditionally only possible with client-side routing.** *(emphasis added)*

This sentence then becomes confusing because the ViewTransitions component in fact does intercept navigation on the page, even though there isn't a full on router à la react-router being provided. Under this alternative but popular definition of client-side router, this sentence would appear to contradict what the component actually provides.

I propose removing the "only possible" phrase, or replacing it with something like "it provides a lightweight router" or "it does this by intercepting navigation".

### 🖥️ Reproduction in StackBlitz (if reporting incorrect content or code samples)

N/A

### PR DESCRIPTION
**Pull request by USKhokhar**
#### What kind of changes does this PR include?
- New or updated content

#### Description

- Closes #3780 
- Updates the opening para for [View-Transition](https://docs.astro.build/en/guides/view-transitions/) for a more clarified explanation, as explained in the listed issue.

Replaces
> Astro provides a routing component that can be added to a single page’s to control page transitions as you navigate away to another page, an effect traditionally only possible with client-side routing. (emphasis added)

with 

> Astro provides a `<ViewTransitions />` routing component that can be added to a single page's `<head>` to control page transitions as you navigate away to another page. It provides a lightweight router that intercepts navigation and allows you to customize the transition between pages. Add this component to a reusable `.astro` component, such as a common head or layout, for animated page transitions across your entire site (SPA mode).

### CODE REVIEW

**Comment by USKhokhar** [comment]
Thanks @sarah11918  for checking out the PR and the response! I get that things are busy, especially with an experimental feature like this. Take your time to review, and I'm here for any more tweaks. Looking forward to your feedback!

File: src/content/docs/en/guides/view-transitions.mdx [hunk 1]
```diff
@@ -7,7 +7,7 @@ i18nReady: true
 
 Support for **opt-in, per-page, view transitions** in Astro projects can be enabled behind an experimental flag. View transitions update your page content without the browser's normal, full-page navigation refresh and provide seamless animations between pages. 
 
-Astro provides a `<ViewTransitions />` routing component that can be added to a single page's `<head>` to control page transitions as you navigate away to another page, an effect traditionally only possible with client-side routing. Add this component to a reusable `.astro` component, such as a common head or layout, for animated page transitions across your entire site (SPA mode).
+Astro provides a `<ViewTransitions />` routing component that can be added to a single page's `<head>` to control page transitions as you navigate away to another page. It provides a lightweight router that intercepts navigation and allows you to customize the transition between pages. Add this component to a reusable `.astro` component, such as a common head or layout, for animated page transitions across your entire site (SPA mode).
```
Comment (ElianCodes) [inline src/content/docs/en/guides/view-transitions.mdx:10]: I think we should lightly mention that the router here is client-side

```suggestion
Astro provides a `<ViewTransitions />` routing component that can be added to a single page's `<head>` to control page transitions as you navigate away to another page. It provides a lightweight client-side router that intercepts navigation and allows you to customize the transition between pages. Add this component to a reusable `.astro` component, such as a common head or layout, for animated page transitions across your entire site (SPA mode).
```

**Review by ElianCodes — COMMENTED** [review]
Hey @USKhokhar, thanks for your PR!

Left a small comment for you to review!

**Comment by USKhokhar** [comment]
Hey @ElianCodes! Thanks for the review. I've made the changes as per the suggestion.

**Review by natemoo-re — APPROVED** [review]
Just double checked with Matthew, this fairly describes our technical implementation! LGTM.

**Pull request merged**